routes/adminroutes.py

```diff
@@ -95,11 +96,10 @@ def get_only_users():
             raise Exception(f"Clerk API error: {response.text}")
             
         users_data = response.json()
-        
-        # Transform user data, filter only users with role 'user'
-        transformed_users = []
-        users_list = users_data  
-        for user in users_list:
+        # Transform user data
```

Comment:
The below three line changes is not a real change (i.e., nothing really changed). But the change in comment reveals a critical misunderstanding of the logic. What we want is a filtered version of users with role 'user.' These are the patients. This is to avoid counting in the users with role 'admin.' These are like doctors and other admins. Your change is simply removing that filtering.